.eslintrc.yaml

```diff
@@ -27,4 +27,5 @@ rules:
   no-restricted-syntax: off
   no-use-before-define: off
   object-curly-newline: off
+  prefer-destructuring: off
```

Comment:
Kept getting a lint error for this rule in metrics.js with no line number. Spent 20 minutes trying to figure out what it was talking about, then gave up and did this.

Reply:
```sh
$ npm run lint

> themesrfun@0.0.22 lint /Users/pdehaan/dev/github/mozilla/ThemesRFun
> eslint --color .

/Users/pdehaan/dev/github/mozilla/ThemesRFun/src/lib/metrics.js
  11:7  error  Use object destructuring  prefer-destructuring
```

It's choking on [/src/lib/metrics.js:11](https://github.com/mozilla/ThemesRFun/pull/89/files#diff-d0154faa85aed1ddcb0a1d33f0484caeR11):

```js
const GA_TRACKING_ID = packageMeta.config.GA_TRACKING_ID;
```

Solution is a bit hacky, but I couldn't figure out how to destructure nested properties on an `import` to get the `GA_TRACKING_ID`:

```js
// Use of package.json for configuration
import { config, extensionManifest, version } from '../../package.json';

import { makeLog } from './utils';
import { selectors } from './store';
import { colorsWithAlpha } from './constants';

const log = makeLog('metrics');

const GA_URL = 'https://ssl.google-analytics.com/collect';
// eslint-disable-next-line prefer-destructuring
const GA_TRACKING_ID = config.GA_TRACKING_ID;
const GA_APP_ID = extensionManifest.applications.gecko.id;
const GA_APP_VERSION = version;
```

Reply:
I can use destructuring like this, but I don't think it's especially better:
```javascript
const {
  version: GA_APP_VERSION,
  config: { GA_TRACKING_ID },
  extensionManifest: {
    applications: {
      gecko: {
        id: GA_APP_ID
      }
    }
  }
} = packageMeta;
```

Reply:
Also just noticed on my screen the error's line number is in black, for some reason:

![capture](https://user-images.githubusercontent.com/21687/36738317-3ac5c9c6-1bab-11e8-87ac-b01d7e6bb698.PNG)

But, since I have a line number now, I'm just going to punt on that lint rule for this chunk.